Timer/src/main/java/com/partnus/timer/TimerType.kt

```diff
@@ -6,4 +6,5 @@ package com.partnus.timer
  */
 enum class TimerType {
     PROGRESS_BAR/*, PROGRESS_RING*/
+    ,PROGRESS_TEXT
 }
```

Comment:
```suggestion
enum class TimerType {
    PROGRESS_BAR,
    PROGRESS_TEXT,
    /*PROGRESS_RING*/
```
styleable에서 progressText가 value=1 이고, progressRing이 value=2 이기 때문에 실수 방지를 위해 PROGRESS_RING 을 마지막으로 보내주시면 좋을 것 같아요!
-> 여기서 주석만 풀어버리면.. ENUM 결과가 PROGRESS_RING이 1이 되고, PROGRESS_TEXT가 2가 되어서 에러가 발생할 것 같아요.. ㅠㅠ